manifests/install.pp

```diff
@@ -319,6 +319,13 @@
         }
       }
 
+      exec { 'create random data for voms-proxy-init':
+        command => '/usr/bin/dd of=/dev/urandom of=/var/lib/cobald/.rnd bs=256 count=1',
```

Comment:
The first `of` should be an `if`. Otherwise this PR looks good to me.

Reply:
Thanks! A second pair of eyes is surely helpful (and of course we should also look at tests later, but only extensive tests actually running the `exec` would have caught this, `puppet-postgresql` is doing such things we might take a look at). 

Should be fixed with the force push from just now :-).